test-crawler/main.go

```diff
@@ -105,20 +106,18 @@ func extractPublicMethodsFromFile(ctx context.Context, filePath string) ([]c.Fun
 }
 
 // makeYAML will make yaml file from public methods.
-func makeYAML(ctx context.Context, filePath string) ([]byte, error) {
+func makeYAML(ctx context.Context, filePath string) error {
 
 	publicMethods, err := extractPublicMethodsFromFile(ctx, filePath)
 	if err != nil {
 		fmt.Print(err)
 		os.Exit(1)
 	}
 
-	var count int
-	for _, fn := range publicMethods {
-		count++
-		fn.ID = count
+	for i, fn := range publicMethods {
+		i++
```

Comment:
incrementing index while iterating trough slice makes no sense and could be dangerous. 
Instead of assigning `fn.Id = I` , generate uuid and assign it to ID.. lately we can generate ID as tuple (functionName + testCase for example)

Reply:
@brdji  I think this is not relative anymore since I changed it. Would love to hear your thoughts.

Reply:
Yep, this has been fixed.